### ISSUE TITLE
Update Demo code

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/PatientLevelPrediction#87]
Work in progress in Document branch.

Updating and expanding the demo functionality.
Includies adding this in the vignettes.

- [x] Add demo learning curve -> done with user input functionality to run in parallel or not.
- [x] Add demo ensemble models
- [x] update demo of singleModel (not working due to code updates)

(DeepLearning and others on hold for next release)

Demo checks should be part of a release milestone!

Added this code in demos to avoid that annoying message that it cannot setwd:

```
currentfftempdir <- getOption("fftempdir")
while (file.access(currentfftempdir)==-1) {
  yourfftempdir <- readline(prompt="Specify a writable fftemp folder:")
  options(fftempdir = yourfftempdir)
  currentfftempdir <- getOption("fftempdir")
}
```

**Comment by PRijnbeek** [comment]
Done in document branch
